pylhc_submitter/autosix.py

```diff
@@ -57,36 +68,40 @@
     default: ``/afs/cern.ch/user/m/mad/bin/madx``
 
 
-- **apply_mad6t_hacks**:
+- **jobid_mask** *(str)*:
 
-    Apply two hacks: Removes '<' in binary call and
-    ignore the check for 'Twiss fail' in the submission file.
-    This hack is needed in case this check greps the wrong lines,
-    e.g. in madx- comments. USE WITH CARE!!
+    Mask to name jobs from replace_dict
 
-    action: ``store_true``
 
+- **max_materialize** *(int)*:
 
-- **jobid_mask** *(str)*:
+    Maximum jobs to be materialized in scheduler. Here: ``None`` leaves the
+    settings as defined in the SixDesk htcondor_run_six.sub template and
+    ``0`` removes it from the template. Warning: This setting modifies the
+     template in the ``sixdesk_directory`` permanently.
+     For more details htcondor API.
 
-    Mask to name jobs from replace_dict
 
+- **max_stage** *(str)*:
 
-- **python3** *(PathOrStr)*:
+    Last stage to be run. All following stages are skipped.
 
-    Path to python to use with sixdb (python3 with requirements
-    installed).
 
-    default: ``python3``
+- **python2** *(PathOrStr)*:
 
+    Path to python to use with run_six.sh (python2 with requirements
+    installed). ONLY THE PATH TO THE DIRECTORY OF THE python BINARY IS
+    NEEDED! And it can't be an Anaconda Distribution.

```

Comment:
I feel like we should explain why it can't be an Anaconda distribution.

Reply:
"Because SixDesk" 
I have no idea why, but it didn't work with anaconda.